ci_stability.sh

```diff
@@ -20,13 +20,24 @@ hosts_fixup() {
     echo "travis_fold:end:hosts_fixup"
 }
 
+install_chrome() {
+    channel=$1
+    deb_archive=google-chrome-${channel}_current_amd64.deb
+    wget https://dl.google.com/linux/direct/$deb_archive
+    sudo dpkg --install $deb_archive || true
+    sudo apt-get install --fix-broken
+    sudo dpkg --install $deb_archive
+}
 
 test_stability() {
     python check_stability.py $PRODUCT
 }
 
 main() {
     hosts_fixup
+    if [ $(echo $PRODUCT | grep '^chrome:') ]; then
+       install_chrome $(echo $PRODUCT | grep --only-matching '\w\+$')
```

Comment:
I guess I might have used two variables rather thing string manipulation, but this is OK for now.